sites/istio.io/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-io
+  namespace: gcsweb
+  annotations:
+    service.beta.kubernetes.io/external-traffic: OnlyLocal
+spec:
+  selector:
+    app: istio-io
+  type: LoadBalancer
+  loadBalancerIP: 35.185.199.142
```

Comment:
Update this with the IP that you created.

Reply:
This is the the one which I created 35.185.199.142